### ISSUE TITLE
WebSocketSubject.observers is always empty

### ISSUE THREAD

**Issue by abarke** [ReactiveX/rxjs#5961]
## Bug Report

**Current Behavior**
`WebSocketSubject.observers` is always empty.

**Expected behavior**
`WebSocketSubject.observers` should contain all subscribers.

**Reproduction**
```js
console.log('always empty', this.webSocketSubject$.observers)
console.log('this works', this.webSocketSubject$._output.observers)
```

**Environment**
- Runtime: [e.g. Node v14.15.1, Chrome v87.0.4280.141]
- RxJS version: 6.6.3
- (If bug is related) Loader, build configuration: webpack

**Possible Solution**
bind `WebSocketSubject._output.observers` to `WebSocketSubject.observers`

**Additional context/Screenshots**

**Comment by benlesh** [comment]
Hi. I'm sorry to tell you this, but `observers` was always meant as an internal API. I think it left "public" > 6 years ago for testing reasons. The way that WebSocketSubject was written it simply doesn't use `observers`.  There's a reason that `observers` is intentionally undocumented, but thank you for bringing this to my attention, as we should definitely privatize it or mark it as internal.

In short, I'm closing this as "won't fix", because we want to stop people from accessing `observers` directly at some point in the near future.
